jannovar-core/src/main/java/de/charite/compbio/jannovar/annotation/AnnotationCollector.java

```diff
@@ -55,8 +55,8 @@
  *
  * Used for the implementation of VariantAnnotator.
  *
- * @author <a href="mailto:peter.robinson@charite.de">Peter N Robinson</a>
- * @author <a href="mailto:max.schubach@charite.de">Max Schubach</a>
+ * @author Peter N Robinson <peter.robinson@charite.de>
+ * @author Max Schubach <max.schubach@charite.de>
```

Comment:
This should be:
```
* @author <a href="mailto:peter.robinson@charite.de">Peter N Robinson</a>
 * @author <a href="mailto:max.schubach@charite.de">Max Schubach</a>
```
and not `Peter N Robinson <peter.robinson@charite.de> ` (wrong javadoc format)